react exercise/users-information/src/pages/userDetails/UserDetails.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from "react";
+import Toggle from "../../components/toggle/Toggle.tsx";
+import Search from "../../components/search/Search";
+import Topper from "../../components/topper/Topper.tsx";
+import CreatorCard from "../../components/creator-card/CreatorCard.tsx";
+import UserTable from "../../components/user-table/UserTable.tsx";
+import { User } from "../../types/user.ts";
+import { FilterButtons } from "../../components/filter-buttons/FilterButtons.tsx";
+import "./UserDetails.scss";
+
+export enum FilterType {
```

Comment:
Most of the times it's enough to define a type that has several "string" options for values

Reply:
For myself: read about the differences between enums and types